benchto-driver/src/main/java/io/trino/benchto/driver/utils/QueryUtils.java

```diff
@@ -130,7 +131,7 @@ private static String resultRowToString(ResultSet resultSet)
                 .map(i -> {
                     try {
                         Object value = resultSet.getObject(i);
-                        return value == null ? "" : value.toString();
+                        return value == null ? "" : (value instanceof byte[]) ? Arrays.toString((byte[]) value) : value.toString();
```

Comment:
nit: nesting ternary operators doesn't look too good, how about splitting this up?
```suggestion
                        if (value == null) {
                             return "";
                        }
                        if (value instanceof byte[]) {
                             return Arrays.toString((byte[]) value);
                        }
                        return value.toString();
```